api-flask/app/database/mixins/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .mysql_primary_key import MysqlPrimaryKeyMixin
```

Comment:
We're using postgres, not mysql, make sure to update the names or it can be misleading